server/src/bin/ingestion-worker.rs

```diff
@@ -1040,13 +1052,6 @@ pub async fn readd_error_to_queue(
             payload.attempt_number
         );
 
-        let _ = redis::cmd("LREM")
```

Comment:
At this spot we should still LREM from processing, since it is going back into the ingestion queue